training/general/src/environments/tensorflow-2.12-cuda11/tests/tensorflow2_12_sample_test_cpu_bert.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Tests running a sample job in the tensorflow 2.12 environment."""
+import os
+import time
+from pathlib import Path
+from azure.ai.ml import command, MLClient
+from azure.ai.ml._restclient.models import JobStatus
+from azure.ai.ml.entities import Environment, BuildContext
+from azure.identity import AzureCliCredential
+
+BUILD_CONTEXT = Path("../context")
+JOB_SOURCE_CODE = "src"
+TIMEOUT_MINUTES = os.environ.get("timeout_minutes", 60)
+STD_LOG = Path("artifacts/user_logs/std_log.txt")
+
+
+def test_tensorflow_2_12():
+    """Tests a sample job using tensorflow 2.12 as the environment."""
+    this_dir = Path(__file__).parent
+
+    subscription_id = os.environ.get("subscription_id")
+    resource_group = os.environ.get("resource_group")
+    workspace_name = os.environ.get("workspace")
+
+    ml_client = MLClient(
+        AzureCliCredential(), subscription_id, resource_group, workspace_name
+    )
+
+    env_name = "tensorflow-2_12-cuda11"
+
+    env_docker_context = Environment(
+        build=BuildContext(path=this_dir / BUILD_CONTEXT),
+        name=env_name,
+        description="Tensorflow 2.12 environment created from a Docker context.",
+    )
+    ml_client.environments.create_or_update(env_docker_context)
+
+    # create the command
+    job = command(
+        code=this_dir / JOB_SOURCE_CODE,  # local path where the code is stored
+        command="pip install -r requirements.txt" \
+                " && python tf_bert.py",
+        environment=f"{env_name}@latest",
+        compute=os.environ.get("intel_cpu_cluster"),
```

Comment:
Are you referring to `compute=os.environ.get("intel_cpu_cluster")` ? This is set locally. I modeled it after the original gpu test script: https://github.com/Azure/azureml-assets/blob/main/training/general/src/environments/tensorflow-2.12-cuda11/tests/tensorflow2_12_sample_test.py where it references `gpu_cluster`

I just changed the name for reference because the oneDNN optimizations needed supported hardware.
Was there a preferred approach to this?

Reply:
gpu_cluster is created as part of below setup of azure resources. You might want to just use 'cpu_cluster' instead of 'intel_cpu_cluster'

https://github.com/Azure/azureml-assets/blob/e2d47987e0154c787ff19f68d65ddca6b2590fab/scripts/setup/create_azure_resources_daily.sh#L7

Reply:
Oh, I see. Thank you for pointing that out. I've made the change to 'cpu_cluster'